components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -343,7 +343,7 @@ protected void findPreviousAuthenticatedSession(HttpServletRequest request,
                     }
 
                     context.setPreviousSessionFound(true);
-                    sequenceConfig = previousAuthenticatedSeq;
+                    sequenceConfig = previousAuthenticatedSeq.cloneObject();
```

Comment:
Thank you Kasun for the advice,
I have fixed this with https://github.com/wso2/carbon-identity-framework/pull/578. 
I have used deep cloning , please review this and advice.